modules/petition.js

```diff
@@ -59,10 +59,12 @@ module.exports = {
                 '(+' + change.toLocaleString('en-GB') + ')',
                 '·',
                 percentageOfElectorate,
+                'of electorate',
+                '(' + domestic.toLocaleString('en-GB') + ')'
```

Comment:
```suggestion
                '(' + domestic.toLocaleString('en-GB') + ')',
```

This breaks the build.